recipes/javascripthon/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "javascripthon" %}
+{% set version = "0.10" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 95fc362624513c5f0ec0ee0f355cf26807621da6fb94af94b0cf0ed313e4e957
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  entry_points:
+    - pj = metapensiero.pj.__main__:main
+
+requirements:
+  host:
+    - python >=3.5
+    - pip
+    - pytest-runner
+  run:
+    - python >=3.5
+    - dukpy
+    - macropy3 =1.1.0b2
```

As well as the `python` change, how about:
```suggestion
    - macropy3 >=1.1.0b2
```

---

I'm not sure, the package's `setup.py` specifically requests this version. Not sure if being more liberal would be such a good idea then?

---

The only downside is that you risk making it impossible to install with other packages (e.g. if something depends on `macropy3 >=1.1`). A few packages can get away with being strict but if too many do it would be impossible to install anything from conda-forge.

It's unlikely to be an issue as not much depends on `macropy3` so it's up to you which you'd prefer.

---

Ok, addressed.